### ISSUE TITLE
Mark tracks as favourite from the app

### ISSUE THREAD

**Issue by kaaholst** [kaaholst/android-squeezer#38]
```
The server tracks "favourite" songs.  The app should expose a UI to (at least) make
a given song as a favourite -- the Android convention would probably be to 'star' it.
```

Original issue reported on code.google.com by `nik@ngo.org.uk` on 2011-09-03 08:07:39

**Comment by kaaholst** [comment]
```
I find this function especially important for radio stations, difficult and slow to
search when not starred.
```

Original issue reported on code.google.com by `paolo.cavallini` on 2015-02-27 07:38:05

**Comment by kaaholst** [comment]
Fixed in 1.5.3

Original issue reported on github.com/nikclayton/android-squeezer by kaaholst on 2020-03-07T19:15:05Z
